cypress/integration/features/task/createAndDeleteTask.feature

```diff
@@ -7,29 +7,55 @@ Feature: To create and delete tasks by the teacher.
     When I go to rooms overview
     And I go to room 'Course with subject and tasks'
     And I click on FAB to create new content
-    And I click on New Task
+    And I click on New Task FAB
     Then I can see create task page
     # And file upload button is disabled
-    When I enter title 'Task Creation and Deletion Test'
+    When I enter title 'Cy Task Creating and Deleting Test'
     And I click on Enable Group Submission
-    And I click on Draft
+    And I click on Draft Checkbox
     And I set task-visibility-start-date to 'today' at '0000'
     And I set task-visibility-due-date to 'tomorrow' at '1000'
     And I enter task description 'Dies ist Deine Aufgabe.'
     And I click on button Submit
     Then I can see room page 'Course with subject and tasks'
-    And I can see task 'Task Creation and Deletion Test'
+    And I can see task 'Cy Task Creating and Deleting Test'
+
+Scenario: Teacher edits and publishes task from room via form
```

Comment:
feature file name can also be updated --> currently it is 'createAndDeleteTask.feature'